### ISSUE TITLE
Set secondary administrator

### ISSUE THREAD

**Issue by oktorok** [vgrem/Office365-REST-Python-Client#294]
Hi, i want to set secondary administrator of all sites, i can do it using requests python but i need to copy manually cookies.

I use to use your module for check values but i don't know if it can modify properties of SharePoint sites and it would be more comfortable for automatization.

Could you share an example or a guide?

Thank you so much

**Comment by vgrem** [comment]
Greetings,

here you go, the example demonstrates  how to add site collection administrator:

```python 

from office365.runtime.auth.user_credential import UserCredential
from office365.sharepoint.client_context import ClientContext

ctx = ClientContext(settings["url"]).with_credentials(
                                             UserCredential("{admin}@{tenant-prefix}.onmicrosoft.com", "{password}"))

new_admin = ctx.web.ensure_user("{username}@{tenant-prefix}.onmicrosoft.com").execute_query()
new_admin.set_property("IsSiteAdmin", True).update().execute_query()


```

**Comment by oktorok** [comment]
Hi vgrem, thank you for the code, but im not sure if is what i want.

I don't want a site collection administrator, i want a secondary admin for a specific site. Normally i would need to give as parameters the siteid and my email.

Maybe is something new in SharePoint, im not sure.

**Comment by vgrem** [comment]
Greetings @oktorok,

to assign secondary site collection administrator you could give it a try a _new_ method available via library

> Note:  since it is not yet available via the latest version (2.2.3) , it needs to be installed from from GitHub:
> 
> `pip install git+https://github.com/vgrem/Office365-REST-Python-Client.git`



`Tenant.set_site_secondary_administrators(site_id, emails)`

where 

- `site_id` - target site id
 - `emails `- the list of site collection emails 

```python

credentials = UserCredential(username, password)
admin_site_url = "https://{0}-admin.sharepoint.com/".format(tenant_prefix)
emails  = ["jdoe", "mdoe@{0}.onmicrosoft.com"]

admin_client = ClientContext(admin_site_url)).with_credentials(credentials)
tenant = Tenant(admin_client)
tenant.set_site_secondary_administrators(site_id=target_site.id, emails=emails).execute_query()
```


[`get_set_site_admin.py` example ](https://github.com/vgrem/Office365-REST-Python-Client/blob/master/examples/sharepoint/tenant/get_site_admin.py 
)
